meos/src/general/span_aggfuncs_meos.c

```diff
@@ -82,7 +82,7 @@ spanset_append_span(SpanSet *ss, const Span *span, bool expand)
   SpanSet *result = spanset_make_exp(spans, ss->count + 1, maxcount,
     NORMALIZE_NO, ORDER);
   pfree(spans);
-  // pfree(ss);
+  pfree(ss);
```

Comment:
This is related to my comment from #531 and is a good change, but we need to specify in the doc of the function that we potentially free the input spanset, so the user always have to call the function as:
```C
ss = spanset_append_span(ss, ...);
```

Also this means we need to revert the changes made to `examples/intspan_agg.c` and `examples/floatspanset_agg.c`